src/BumpChart.js

```diff
@@ -60,7 +61,7 @@ export default class BumpChart extends Plot {
       tickFormat: val => {
         const data = this._formattedData;
         const xDomain = this._xDomain;
-        const endData = data.filter(d => d.x === xDomain[xDomain.length - 1]);
+        const endData = data.filter(d => this._time ? d.x.getTime() === xDomain[xDomain.length - 1].getTime() : d.x === xDomain[xDomain.length - 1]);
```

Comment:
instead of relying on `this._time` to be true (which may not always relate to the X-axis), let's do a more manual Date detection (and only convert the min and max values once, not in each loop of the filter). Something like this:

```js
const xDomain = this._xDomain;
const xMin = xDomain[0] instanceof Date ? xDomain[0].getTime() : xDomain[0];
const startData = data.filter(d => (d.x instanceof Date ? d.x.getTime() : d.x) === xMin);
```